src/cpp/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ function(ov_genai_build_jinja2cpp)
         set(JINJA2CPP_WITH_JSON_BINDINGS "none" CACHE STRING "")
         set(JINJA2CPP_STRICT_WARNINGS OFF CACHE BOOL "")
         set(JINJA2CPP_PIC ON CACHE BOOL "")
+        set(RapidJSON_DIR "${FETCHCONTENT_BASE_DIR}/rapidjson-build")
```

Comment:
Is is caused by `jinja2cpp-src/thirdparty/internal_deps.cmake` calling `find_package(RapidJSON REQUIRED)` isntead of `add_subdirectory()`?

How would that cause picking up RapidJSON from another build?

Reply:
> RapidJSON from another build

I don't know :) but now, it finds proper RapidJson folder